config.json

```diff
@@ -5,8 +5,9 @@
     "distance_units": "statute", 
     "flip_horizontal": false, 
     "flip_vertical": false, 
-    "ownship": "N701GV", 
+    "ownship": "N129BZ", 
```

Comment:
We should probably come up with a good stand-in for the ownship, and potentially figure out a way to leave this file in the REPO, but not commit the ownship changes.

Reply:
Would that be part of the configuration enhancement, like a web-based setup of preferences, values, etc?